src/main/webapp/script.js

```diff
@@ -30,5 +31,6 @@ function displayLocation(position) {
 }
 
 function displayError() {
-  document.getElementById("location").innerHTML = "please enter your zipcode";
+  // TODO: Write error to user interface
+  console.log("Error - please enter zipcode");
```

Comment:
"please enter zipcode" is again repetitive here, possibly try to use literal variable for this. Since you deleted the previous code.

Reply:
I updated the error message to be more specific - it's not user facing so I just described the issue instead of an action

Reply:
I was told it is good to never have large strings, assigned throughout the code, instead assign that string to a variable in the beginning of the code. Does this make sense? For example, when you are making a URL API call you should make a variable that you assign that string URL, rather then just writing that string throughout your code.

Reply:
I factored out the alert messages to a string literal. For now though, I  think I am going to keep the console messages inline because there are only used once and will likely become more specific. I don't think that they disrupt the flow of code too much.